includes/specials/SMW_SpecialAsk.php

```diff
@@ -492,7 +491,7 @@ protected function getInputForm( $printoutstring, $urltail, $navigation = '', $d
 		}
 
 		$result .= '}}</textarea></div><p></p>';
-		$result .= ( $navigation !== '' ? '<p>'. $sarchInfoText . '</p>' . '<hr class="smw-form-horizontalrule">' .  $navigation . '&#160;&#160;&#160;' . $downloadLink : '' ) . "</fieldset>\n";
+		$result .= ( $navigation !== '' ? '<p>'. $sarchInfoText . '</p>' . '<hr class="smw-form-horizontalrule">' .  $navigation . '&#160;&#160;&#160;' . $downloadLink : '' ) . "\n";
```

Comment:
There is a small issue with the fieldset, without

![image](https://cloud.githubusercontent.com/assets/1245473/15270161/befc7da0-1a15-11e6-8e68-52aca82c782a.png)

with the PR

![image](https://cloud.githubusercontent.com/assets/1245473/15270158/a25a5f00-1a15-11e6-8183-e9e8ef62a8b7.png)